### PR TITLE
Upstream cherry pick fixes 0811

### PR DESCRIPTION
**Pull request by jjsjann123**
cherry-pick patches for issues found in upstream CI.

### CODE REVIEW

**Review by kevinstephano — APPROVED** [review]
LGTM

**Comment by jjsjann123** [comment]
Tests passed locally, I'll merge once lintrunner finishes

File: torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp [hunk 1]
```diff
@@ -572,7 +572,7 @@ void scheduleTranspose(Fusion* fusion, TransposeParams params) {
   //    t0
   //    |
   //   cache
-  //   /  \
+  //   |  |
```
Comment (naoyam) [inline torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp:575]: Does this trigger something?

Comment (jjsjann123) [reply torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp:575]: Yep it did. upstream linter sees `\` at the end and thought we are doing multiple line comment.... :rofl:

Comment (zasdfgbnm) [reply torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp:575]: I hate linter 😤

Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp:575]: wow

**Pull request merged**